library/examples/thing.json

```diff
@@ -0,0 +1,6 @@
+{
+  "kind": "thing",
```

Comment:
Not sure this `examples` directory is useful? Or the two files should be updated to:
- contain array? Or do we want a single object to illustrate the type?
- contain correct values: `source` shouldn't be an object, and `howMuch` doesn't exist

Reply:
Let's remove it, I only created it to draft examples in the beginning